src/index.ts

```diff
@@ -171,6 +171,7 @@ const graphqlServer = graphqlHTTP((req, res, params) => {
     userAgent,
     appToken,
     xOriginalSessionID,
+    isMutation: req.body.query.includes("mutation"),
```

Comment:
This is the place where all the loaders are being created, and so it seemed like a good place to inject this, and we have access to the whole request but its a vanilla `Request` object, so we don't get any built-in parsing (yet), so this is the best we can do (besides try to parse the query better, _or_ inject the option somewhere else, although I couldn't think of a better place).

Reply:
Love it